GitCommands/Git/GitModule.cs

```diff
@@ -2103,6 +2103,14 @@ public string CommitCmd(bool amend, bool signOff = false, string author = "", bo
                 command += " --author=\"" + author + "\"";
             }                
 
+            if (gpgSign)
+            {
+                command += " -S";
+
+                if (!string.IsNullOrEmpty(gpgKeyId))
+                    command += gpgKeyId;
```

Comment:
are we missing a space here?

Reply:
From [git scm page](https://git-scm.com/docs/git-commit#git-commit--Sltkeyidgt), a space must not be there.

Reply:
all good then, thank you